packages/composites/anchor/README.md

```diff
@@ -6,18 +6,21 @@ This composite adapter fetches the price of a given token from the Anchor protoc
+|    ✅     |        `COLUMBUS_5_LCD_URL`        | The URL to a Terra `columbus-5` full node to query on-chain mainnet data. At least 1 of 3 LCD_URLs must be provided. |                                             string                                              |                                                |     |
+|    ✅     |        `BOMBAY_12_LCD_URL`         | The URL to a Terra `bombay-12` full node to query on-chain testnet data. At least 1 of 3 LCD_URLs must be provided.  |                                             string                                              |                                                |     |
+|    ✅     |        `LOCALTERRA_LCD_URL`        |   The URL to a locally running Terra full node to query on-chain data. At least 1 of 3 LCD_URLs must be provided.    |                                             string                                              |                                                |     |
+|           |         `DEFAULT_CHAIN_ID`         |                               The default `chainId` value to use as an input parameter                               |                                             string                                              |                  `columbus-5`                  |
+|           |  `ANCHOR_VAULT_CONTRACT_ADDRESS`   |                                       The address of the Anchor Vault contract                                       | Address can be found [here](https://docs.anchorprotocol.com/smart-contracts/deployed-contracts) |  `0xA2F987A546D4CD1c607Ee8141276876C26b72Bdf`  |
+|           | `TERRA_BLUNA_HUB_CONTRACT_ADDRESS` |                                        The address of bLuna contract in Terra                                        | Address can be found [here](https://docs.anchorprotocol.com/smart-contracts/deployed-contracts) | `terra1mtwph2juhj0rvjz7dy92gvl6xvukaxu8rfv8ts` |
+|           |   `STETH_POOL_CONTRACT_ADDRESS`    |                                            The address of stEth contract                                             |                   Address can be found by finding the stETH/ETH pool in Curve                   |  `0xdc24316b9ae028f1497c275eb9192a3ea0f67022`  |
+|           |     `LUNA_TERRA_FEED_ADDRESS`      |                                           The Terra address of `LUNA` feed                                           |        Address can be found by here https://docs.chain.link/docs/terra/data-feeds-terra/        | `terra1gfy9nxj2xwd4vcupzfelk34u3qjkvp3vcjveg6` |
+|           |      `ETH_TERRA_FEED_ADDRESS`      |                                           The Terra address of `ETH` feed                                            |        Address can be found by here https://docs.chain.link/docs/terra/data-feeds-terra/        | `terra1a39jndcuh64ef2qzt5w8mh46m5ysc34a9qd2e5` |
```

Comment:
We should also have env vars for the number of decimals on these feeds, right?

Reply:
Won't it always be 8?  How likely is it for a feed to change the number of DPs?

Reply:
Hmm I guess it's very unlikely the feeds will change. Guess we can keep them as-is